docs/source/contributing.rst

```diff
@@ -200,26 +200,37 @@ Function and object doc strings
 For inline strings documenting functions and objects ("doc strings"), we use the
 `numpydoc style (Numpy docstring format) <https://numpydoc.readthedocs.io/en/latest/format.html>`_.
 
-Sphinx ReadTheDocs documentation
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Jupter Book ReadTheDocs documentation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Echopype documentation (`<https://echopype.readthedocs.io>`_) is based on `Jupyter Book <https://jupyterbook.org/en/stable/intro.html>`_
+which are rendered under the hood with `Sphinx <https://www.sphinx-doc.org>`_. The documentation is hosted at
+`Read The Docs <https://readthedocs.org>`_. The documentation package dependencies are found
+in the ``docs/requirements.txt`` file, and the source documentation files are in the ``docs/source`` directory. The echopype development
+conda environment will install all required dependencies.
+
+Our documentation are currently a mixture of the following file formats:
+
+- `CommonMark <https://commonmark.org/>`_ and `MySt <https://jupyterbook.org/en/stable/content/myst.html>`_ Markdown
+- `Jupyter Notebook <https://jupyter-notebook.readthedocs.io/en/latest/notebook.html>`_
+- `reStructuredText <https://docutils.sourceforge.io/rst.html>`_
 
-Echopype documentation (`<https://echopype.readthedocs.io>`_) is based on
-`Sphinx <https://www.sphinx-doc.org>`_ and is hosted at
-`Read The Docs <https://readthedocs.org>`_. The sphinx files are found
-in the ``docs`` directory, and the source documentation files, written in
-`reStructuredText <https://www.sphinx-doc.org/en/main/usage/restructuredtext/index.html>`_
-(``.rst``) format, are in the ``docs/source`` directory. The echopype development
-conda environment will install all required Sphinx dependencies.
-To run Sphinx locally:
+To run Jupyter Book locally:
 
 .. code-block:: bash
 
-    cd docs
-    sphinx-build -b html -d _build/doctrees source _build/html
+    jupyter-book build docs/source --path-output docs
 
-To view the generated HTML files generated by Sphinx, open the
+To view the generated HTML files generated by Jupyter Book, open the
 ``docs/_build/html/index.html`` in your browser.
 
+Jupyter Book `configurations <https://jupyterbook.org/en/stable/customize/config.html>`_ can be found in the `docs/source/_config.yml` file.
+The `table of contents <https://jupyterbook.org/en/stable/structure/toc.html>`_ arragements for the sidebar can be found in ``docs/source/_toc.yml`` file.
+
+When ready to commit your changes, please pull request your changes to the `stable` branch. Once the PR is submitted, the `pre-commit` CI will run for basic spelling and formatting check (See the `pre-commit hooks section <contributing.html#pre-commit-hooks>`_ for more details). Any changes from the `pre-commit` check have to be pulled to your branch (via `git pull`) before your push further commits. You will also be able to view the new built doc in the PR via the "docs/readthedocs.org:echopype" entry shown below.
```

Comment:
my only comment, otherwise i think this is good to go!

```suggestion
When ready to commit your changes, please pull request your changes to the `stable` branch. Once the PR is submitted, the `pre-commit` CI will run for basic spelling and formatting check (See the `pre-commit hooks section <contributing.html#pre-commit-hooks>`_ for more details). Any changes from the `pre-commit` check have to be pulled to your branch (via `git pull`) before your push further commits. You will also be able to view the newly built doc in the PR via the "docs/readthedocs.org:echopype" entry shown below.
```